README.md

```diff
@@ -1,77 +1,67 @@
+# Bowling Challenge
```

Comment:
Including a section on "how to install" and "how to use" might be helpful for anyone cloning the repo.